app/controllers/container_route_controller.rb

```diff
@@ -11,6 +11,13 @@ def show_list
     process_show_list
   end
 
+  # handle buttons pressed on the button bar
+  def button
```

Comment:
Please check if you can reuse the button method via the mixin as well, they all look very similar except for the entity type which can be passed as a parameter to it.